almanach/common/AlmanachException.py

```diff
@@ -12,5 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+
```

Comment:
Well of course but since file naming convention is part of PEP8 and this PR is all about PEP8 I would change that right away.

https://www.python.org/dev/peps/pep-0008/#package-and-module-names